django/db/models/sql/query.py

```diff
@@ -82,6 +82,11 @@ def get_children_from_q(q):
 class RawQuery:
     """A single raw SQL query."""
 
+    has_select_fields = True
+    contains_aggregate = []
```

Comment:
```suggestion
    contains_aggregate = False
```
All other `contains_aggregate` attributes are boolean